.github/workflows/continuous-integration.yml

```diff
@@ -49,15 +49,11 @@ jobs:
           coverage: "pcov"
           ini-values: "zend.assertions=1"
 
-      - name: "Cache dependencies installed with composer"
-        uses: "actions/cache@v2"
+      - name: "Install dependencies with Composer"
+        uses: "ramsey/composer-install@v1"
         with:
-          path: "~/.composer/cache"
-          key: "php-${{ matrix.php-version }}-composer-locked-${{ hashFiles('composer.lock') }}"
-          restore-keys: "php-${{ matrix.php-version }}-composer-locked-"
-
-      - name: "Install dependencies with composer"
-        run: "composer update --no-interaction --no-progress"
+          dependency-versions: "highest"
+          composer-options: "--prefer-dist"
```

Comment:
I assume this is an additional change to set `--prefer-dist`.

Reply:
You're right, I did not notice that, and I don't think it is useful here, let me remove it.